retail/interactive-tutorial/RetailSearch.Samples.Tests/SearchWithBoostSpecTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Linq;
+using Xunit;
+
+namespace RetailSearch.Samples.Tests
+{
+    public class SearchWithBoostSpecTest
+    {
+        [Fact]
+        public void TestSearchWithBoostSpec()
+        {
+            const string ExpectedProductTitle = "Tee";
+
+            var response = SearchWithBoostSpecTutorial.Search();
+
+            var actualProductTitle = response.ToArray()[0].Results[0].Product.Title;
+
+            Assert.Contains(ExpectedProductTitle, actualProductTitle);
```

Comment:
Similar suggestion here as for the other sample. Use the Assert on the collection or IEnumerable, as if the  tests failes, the failures are clearer.

Reply:
Added assert for collection.